php/tests/proto/test_service.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package foo;
+
+option php_generic_services = true;
```

Comment:
Is `generic service` part of protobuf terminology? This term is not used in PHP, so it would make more sense from a PHP perspective to make this option `php_interfaces`